topics/icrc_29_window_post_message_transport.md

```diff
@@ -16,10 +16,11 @@ For this standard to represent an [ICRC-25](https://github.com/dfinity/wg-identi
 * The user's machine is not compromised. In particular, the browser must deliver messages unchanged and represent the `origin` of the different parties correctly.
 * DNS entries are not compromised. The user's machine must be able to resolve the domain names of the relying party and signer correctly.
 
-## Establishing a Communication Channel
+## Communication Channel
 
-A [window.postMessage](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage) communication channel is initiated by the relying party. The relying party opens a new window and polls repeatedly for the signer to reply with a message indicating that it is ready for interactions.
-The message sent by the relying party has `targetOrigin` set to `'*'` and is a [JSON-RPC 2.0](https://www.jsonrpc.org/specification) call with method `icrc29_status` and no parameters:
+The relying party initiates and maintains the communication channel by opening a new window for the signer and periodically sending `icrc29_status` messages using the [window.postMessage](https://developer.mozilla.org/en-US/docs/Web/API/Window/postMessage) API.
```

Comment:
We loose the notion that the signer is supposed to reply with a message indicating that it is ready for interactions. Is this expected?

Reply:
You're right, added it back